go-controller/pkg/util/kube.go

```diff
@@ -716,7 +716,7 @@ func ExternalIDsForObject(obj K8sObject) map[string]string {
 	if gk.String() == "" {
 		kinds, _, err := scheme.Scheme.ObjectKinds(obj)
 		if err != nil || len(kinds) == 0 || len(kinds) > 1 {
-			klog.Warningf("BUG: object has no / ambiguous GVK: %#v, err", obj, err)
+			klog.Warningf("BUG: object %v has no / ambiguous GVK: %#v, err", obj, err)
```

Comment:
`Object %v has missing or ambiguous GVK: %v`

Reply:
yup deff no need for BUG, removed

Reply:
also intended to remove the `/`, if you have the chance, replace it with `or`. Trying to improve readability.

Reply:
agreed very much hits eye.

Reply:
done.